snapcraft.yaml

```diff
@@ -1421,6 +1421,22 @@ parts:
       git cherry-pick -x c35ac36962dd5cf691ff2b464055648ff8bcf463 # lxd/device/nic_ovn: Only stop device if network is populated
       git cherry-pick -x 551c252d1653360ccfda981b7a9a42fc8eed5370 # lxd/storage/drivers: Always use default block.filesysem for VM volumes
       git cherry-pick -x 6ae9c4e522c226efed94af447dd9e92ef658e28a # doc: pin version of myst-parser
+      git cherry-pick -x a6e08cc8b897ea149ee5fa395cb9e242749f7db1 # lxd/storage/drivers/zfs: Round to zfs.blocksize or 16KiB
+      git cherry-pick -x 33d712497bfc5d6a8892bd8f6e1ddb3583bee1eb # lxd/storage/drivers: Refactor volume size rounding logic
```

Comment:
This one doesn't cherry-pick cleanly: https://launchpadlibrarian.net/738655677/buildlog_snap_ubuntu_focal_amd64_lxd-5.0-candidate_BUILDING.txt.gz:

```
+ git cherry-pick -x 33d712497bfc5d6a8892bd8f6e1ddb3583bee1eb
Auto-merging lxd/storage/drivers/utils.go
Auto-merging lxd/storage/drivers/driver_zfs.go
Auto-merging lxd/storage/drivers/driver_lvm.go
CONFLICT (content): Merge conflict in lxd/storage/drivers/driver_lvm.go
Auto-merging lxd/storage/drivers/driver_common.go
CONFLICT (content): Merge conflict in lxd/storage/drivers/driver_common.go
error: could not apply 33d712497... lxd/storage/drivers: Refactor volume size rounding logic
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
```

Reply:
hrm it did locally

Reply:
https://github.com/canonical/lxd-pkg-snap/pull/498/files should fix it